scripts/exportAssets.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+import os
+import sys
+import argparse
+import json
+from util import ElasticsearchUtil
+from util import Logger
+from util import Utility
+
+EXPORT_LOG = "/tmp/ExportAssets.log"
+
+esUtil = ElasticsearchUtil(EXPORT_LOG)
+logger = Logger(log_file=EXPORT_LOG)
+logging, rotating_handler = logger.configure_and_return_logging()
+UTIL = Utility(log_file=EXPORT_LOG)
+
+OUTPUT_DIR = os.path.dirname(os.path.realpath(__file__)) 
+
+INDEX = esUtil.KIBANA_INDEX
+TYPE = None
+ID = None
+
+DASHBOARD = "dashboard"
+VISUALIZATION = "visualization"
+SEARCH = "search"
+
+TO_FILE = {}
```

Comment:
It will eventually become a dictionary using a full file path as a key and the content to be written to that file as the value. The name made sense in the moment but I see now that it is confusing. I'll change it to something more descriptive